notebooks/data-sources/TestGrid/testgrid_EDA.ipynb

```diff
@@ -7,7 +7,7 @@
     "# TestGrid: initial EDA and data collection \n",
```

Comment:
<p>@csibbitt this is great. Certainly removes the ambiguity of the year of collection. </p><p> </p><p>But I think it's best to tackle this across a separate issue, because as format is also used across several other metric collection notebooks where this change needs to be reflected else they'd break.</p><p> </p><p>Such as cell [3] in this https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/notebooks/data-sources/TestGrid/metrics/build_pass_failure.ipynb (and the other notebooks in this directory too) where it expects data of the old format.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
yes there is some automation code across the notebooks in the [metrics directory](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/notebooks/data-sources/TestGrid/metrics/) where the saved dashboard data from this notebook is being read for metric calculation steps.

> It's not just the lack of the year, the other scheme is ambiguous

I think the format generated by `date.isoformat()` ie. `testgrid_yyyy-mm-dd.json` will remove this ambiguity and that should be updated across all the notebooks where this collected data is being referenced.

so yes, maybe lets tackle this in a separate PR/issue?